ivre/zgrabout.py

```diff
@@ -236,6 +243,29 @@ def zgrap_parser_http(data, hostrec):
             })
             banner += (b"Server: " + utils.nmap_decode_data(server[0]) +
                        b"\r\n\r\n")
+        if headers.get('www_authenticate'):
+            auths = headers.get('www_authenticate')
+            for auth in auths:
+                if auth[:4].lower() == 'ntlm' and auth[5:]:
+                    msg = utils.decode_b64(auth[4:].strip().encode())
+                    msg = utils.ntlm_extract_info(msg)[5:]
+                    infos = {k: (int(v) if k == 'ntlm-version' else
+                                 utils.decode_b64(v.encode()).decode())
+                             for k, v in (item.split(':', 1)
+                                          for item in msg.split(','))}
+
+                    keyvals = zip(ntlm_values,
+                                  [infos[k] if k in infos.keys() else ''
+                                   for k in ntlm_keys])
```

Comment:
```suggestion
                    keyvals = zip(ntlm_values, [infos.get(k, '') for k in ntlm_keys])
```
Or even better (but you need to adapt the code below:
```suggestion
                    keyvals = zip(ntlm_values, [infos.get(k) for k in ntlm_keys])
```